infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/config/LogReplicationFullTableConfig.java

```diff
@@ -0,0 +1,37 @@
+package org.corfudb.infrastructure.logreplication.config;
+
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+import org.corfudb.infrastructure.ServerContext;
+import org.corfudb.runtime.LogReplication.LogReplicationSession;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+
+
+/**
+ * This class represents any Log Replication Configuration,
+ * i.e., set of parameters common across all Clusters.
+ */
+public class LogReplicationFullTableConfig extends LogReplicationConfig {
+
+    /**
+     * Set of streams to drop on Sink if streams to replicate info differs from Source, which should only happen
+     * when Source and Sink are on different versions.
+     */
+    @Getter
+    @Setter
+    private Set<UUID> streamsToDrop;
```

Comment:
This field tracks the streams which are considered as 'not to be replicated'.  In case of the Logical Group model, it can have the streams which do not have the 'group' option set.

We will have to define this behavior a bit more for other replication models as the streams to replicate can differ between versions.  Source and Sink can be on different versions in all models.